mgmtsystem/openshift.py

```diff
@@ -100,3 +100,10 @@ def list_template(self):
             entity = Template(meta['name'], meta['namespace'])
             entities.append(entity)
         return entities
+
+    def list_image_openshift(self):
+        entities = []
+        entities_j = self.o_api.get('image')[1]['items']
+        for entity_j in entities_j:
+            entities.append(entity_j['metadata'])
```

Comment:
@jaryn Initially I wrote it so it would return the kubernetes image instance, but it didn't serve the purpose cause the kubernetes Image instance does not return the "@sha" name that could be used for comparison with another list.